frontend/packages/llama-stack-modular-ui/frontend/src/index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
-<html lang="en-US" class="pf-v6-theme-dark">
+<html lang="en-US">
 
 <head>
   <meta charset="utf-8">
-  <title>Patternfly Seed</title>
-  <meta id="appName" name="application-name" content="Patternfly Seed">
+  <title>Llama Stack Modular UI</title>
```

Comment:
We are in the process of renaming the playground to `RAG Playground` , let's add this to the list of renamings.